chevah/empirical
# [#2897] Fix test-timer to proxy attributed of the wrapped method.

Problem
======

When test-timer is used to measure the exact test time (without setup/teardown) it wraps the test method.

This wrapper does not export the attributes attached to the wrapped method

Changes
=======

Use a smarter wrapper which proxies the attributes.

As a drive by I have updated the release notes of an older version.


How to test
=========

reviewers: @alibotean 

check that changes make sense.

run test with the test timer plugin:

`paver test normal --with-timer`

To fix this I wrote the test first... so you can revert the changed from the main code and keep the tests.

When you run the test with `--with-timer` the test should fail with the old code and pass with the new code.
